src/Components/SearchAutoSuggest/AdditionalContext/Grids/Artist.tsx

```diff
@@ -0,0 +1,82 @@
+import { ArtistSearchAdditionalContext_artist } from "__generated__/ArtistSearchAdditionalContext_artist.graphql"
```

Comment:
In this folder, we can create all the components we need for the different 'contexts' for the RHS. So, one person can be working on the artist case (show collections OR artworks), one person can be working on the partner case (show rep artists), one person can be working on the sale case, and so on. If we split up those cases, we'll all be working on separate files and can parallelize some of that work among 4 of us (vs just 2).